### ISSUE TITLE
Change of Dropdown selection throws error: Cannot read properties of undefined (reading 'length') primeng dropdown

### ISSUE THREAD

**Issue by RupenAnjaria** [primefaces/primeng#14007]
### Describe the bug

When a dropdown is changed the selection doesn't result any change and throws error in the console which reads as:

core.mjs:10592 ERROR TypeError: Cannot read properties of undefined (reading 'length')
    at get inputClass [as inputClass] (primeng-dropdown.mjs:628:135)
    at _class2_span_2_Template (primeng-dropdown.mjs:73:84)
    at executeTemplate (core.mjs:11986:13)
    at refreshView (core.mjs:13481:13)
    at detectChangesInView (core.mjs:13646:9)
    at detectChangesInEmbeddedViews (core.mjs:13589:13)
    at refreshView (core.mjs:13505:9)
    at detectChangesInView (core.mjs:13646:9)
    at detectChangesInComponent (core.mjs:13621:5)
    at detectChangesInChildComponents (core.mjs:13659:9)

### Environment

Angular: 16.2.0
PrimeNG: 16.7.0
Browser: Chrome (Latest)

### Angular version

16.2.0

### PrimeNG version

16.7.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.13.0

### Browser(s)

Chrome

### Steps to reproduce the behavior

Create a dropdown.
Prefill it's values.
Try changing it.

**Comment by ibrahimkelly** [comment]
Also found it.

**Comment by Hambardzum93** [comment]
I also faced to this issue

**Comment by chillapp** [comment]
Me too

**Comment by benjaminknauer** [comment]
I also face the issue. Did you guys found a workaround yet?

It can be reproduced with a simple Dropdown like this. PLEASE fix or rollback immediately.

```
<p-dropdown
  [options]="[{ value: 'NY', label: 'New York' }]"
  optionValue="value"
  optionLabel="label"
></p-dropdown>
```

**Comment by adcore-dev** [comment]
I think it's a same issue:
https://github.com/primefaces/primeng/issues/13996
All this happen after latest changes in 16.7.0

**Comment by cetincakiroglu** [comment]
Hello everyone,

With version 16.7.0, we have released the first part of the accessibility implementation for inputs that we have been working on for a long time. Based on the research we conducted during the implementation, we decided that many components needed changes in their structure and simplification, and these components were rewritten. Unfortunately, after extensive testing, there may be unexpected and unforeseen bugs that have emerged. We are grateful for your reports, and we are working to address them in the upcoming release!
